tika-parsers/src/main/resources/META-INF/services/org.apache.tika.detect.Detector

```diff
@@ -14,4 +14,4 @@
 #  limitations under the License.
 
 org.apache.tika.parser.microsoft.POIFSContainerDetector
-org.apache.tika.parser.pkg.ZipContainerDetector
+org.apache.tika.parser.pkg.ZipContainerDetector
```

Comment:
Remove this change. I guess, your editor has removed trailing newline, check for some configuration option to avoid it.

Also, you can use `git add -p` to choose what to commit instead of committing all changes.